openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/EntitySummaryPanel.style.less

```diff
@@ -37,6 +37,13 @@
   -ms-overflow-style: none;
   scrollbar-width: none;
 
+  .summary-panel-statistics-count {
+    color: #37352f;
+    font-size: 24px;
+    line-height: 32px;
+    font-weight: 600;
+  }
+
```

Comment:
We should check relevant font present in Typography.Text and if not found then we should try for custom css

Reply:
Got it. I will check that and update this.